schema/defs-windows.json

```diff
@@ -0,0 +1,20 @@
+{
+    "definitions": {
+        "cpuShares": {
+            "description": "Relative weight to other containers with CPU Shares defined",
+            "type": "integer",
+            "minimum": 0,
```

Comment:
And they're right, this is incorrect. Thanks for catching. Updated in 6c66d16ea27afcbcdbf2aa35242a497377d4198f